packages/tradershub/src/components/DemoRealSwitcher/DemoRealSwitcher.tsx

```diff
@@ -0,0 +1,78 @@
+// TODO: Create a Generic component for this once the designed is finalized.
+import React, { useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { LabelPairedChevronDownSmRegularIcon } from '@deriv/quill-icons';
+import { Listbox, Transition } from '@headlessui/react';
+
+const accountTypes = [
+    { label: 'Demo', value: 'demo' },
+    { label: 'Real', value: 'real' },
+];
+
+const DemoRealSwitcher = () => {
+    const [selected, setSelected] = useState(accountTypes[0]);
+    const { label, value } = selected;
+
+    return (
+        <div className='w-auto'>
+            <Listbox as='div' onChange={setSelected} value={selected}>
+                {({ open }) => (
+                    <div className='relative mt-1'>
+                        <Listbox.Button
+                            as='button'
+                            className={`cursor-pointer relative w-full py-[3px] px-400 border-75 rounded-200 flex gap-x-400 items-center ${
+                                value === 'demo' ? 'border-status-light-information' : 'border-status-light-success'
+                            }`}
+                        >
+                            <Text
+                                bold
+                                className={
+                                    value === 'demo' ? 'text-status-light-information' : 'text-status-light-success'
+                                }
+                                size='sm'
+                            >
+                                {label}
+                            </Text>
+                            <LabelPairedChevronDownSmRegularIcon
+                                className={qtMerge(
+                                    'transform transition duration-200 ease-in-out',
+                                    value === 'demo' ? 'fill-status-light-information' : 'fill-status-light-success',
+                                    open && ' -rotate-180'
```

Comment:
Can use cva here and make a className as it will be more readable.

Reply:
my answer is still the same here